integration/tsconfig-test.ts

```diff
@@ -27,7 +28,7 @@ const DEFAULT_CONFIG = {
     strict: true,
     target: "ES2019",
     baseUrl: ".",
-  },
+  } as Omit<TsConfigJson.CompilerOptions, "plugins">,
```

Comment:
Wouldn't it be better to type `DEFAULT_CONFIG` instead?

Reply:
sure, I can do that, it might just be a smidge more verbose since I have to exclude the nested `plugins` which have `unknown` properties and make the whole thing not JSON.stringifiable

Reply:
Done